evm/src/executor/backend/mod.rs

```diff
@@ -1281,6 +1338,13 @@ impl Default for BackendInner {
             next_fork_id: Default::default(),
             persistent_accounts: Default::default(),
             precompile_id: revm::precompiles::SpecId::LATEST,
+            // grant the cheatcode,default test and caller address access to execute cheatcodes
+            // itself
+            cheatcode_access_accounts: HashSet::from([
+                CHEATCODE_ADDRESS,
+                TEST_CONTRACT_ADDRESS,
+                CALLER,
```

Comment:
same here technically

Reply:
the caller is also set separately, so this is also technically redundant, but also doesn't hurt